src/Data/List/Membership/Propositional/Properties.agda

```diff
@@ -331,16 +332,17 @@ finite inj (x ∷ xs) fᵢ∈x∷xs = excluded-middle helper
     lemma i≤j i≰1+j refl = i≰1+j (≤-step i≤j)
 
     f′ⱼ∈xs : ∀ j → f′ j ∈ xs
-    f′ⱼ∈xs j with i ≤? j
-    ... | yes i≤j = ∈-if-not-i (<⇒≢ (s≤s i≤j))
-    ... | no  i≰j = ∈-if-not-i (<⇒≢ (≰⇒> i≰j) ∘ sym)
+    f′ⱼ∈xs j with i ≤ᵇ j | Reflects.invert (≤ᵇ-reflects-≤ i j)
```

Comment:
Hmm I'm not sure I quite understand the purpose of this change. Yes, `i ≤ᵇ j` is more efficient to pattern match on than `i ≤? j`, but don't we lose all the benefits when we call `Reflects.invert (≤ᵇ-reflects-≤ i j)` which presumably has to reconstruct the unary proof?

If there is a purpose for the change, are you proposing that all uses of `i ≤? j` should be replaced by `Reflects.invert (≤ᵇ-reflects-≤ i j)`

Reply:
Without this change there is a type error.
We did not go hunting for uses of `i ≤? j`, the build failed and
@Taneb fixed it.